conans/util/env_reader.py

```diff
@@ -9,8 +9,11 @@
 import os
 
 
-def get_env(env_key, default=None, environment=os.environ):
+def get_env(env_key, default=None, environment=None):
```

Comment:
Yeah, I was about to tell you to change the parameter

Reply:
Done!!! Let's see the CI

Reply:
@lasote thanks for the fix!!! 
As a reference: http://effbot.org/zone/default-values.htm